src/yape06/yape06.h

```diff
@@ -55,11 +55,14 @@ class Yape06 {
     compute_laplacian<u_char, int>(srd_d, laplacian, w, h, Dxx, Dyy, sx, sy, ex,
                                    ey);
 
-    row = (sy * w + sx) | 0;
+    //row = (sy * w + sx) | 0;
+    row = (sy * w + sx);
     for (y = sy; y < ey; ++y, row += w) {
       for (x = sx, rowx = row; x < ex; ++x, ++rowx) {
 
         lv = laplacian[rowx];
+        //JSLOGi("lv is %d", lv);
+        //std::cout << "lv is: " << lv << std::endl;
```

Comment:
If we print values from `lv` we get always zero that is not correct. If we test the jsfeatNext code and print the lv values we get always an integer (positive or negative). This is the source of the issue.